workspaces/liferay-sample-workspace/client-extensions/liferay-sample-editor-config-contributor/src/index.ts

```diff
@@ -9,6 +9,33 @@ import {
 } from '@liferay/js-api/editor';
 
 const editorConfigTransformer: EditorConfigTransformer<any> = (config) => {
+
+	// Alloy Editor
+
+	const toolbars: any = config.toolbars;
+
+	if (typeof toolbars === 'object') {
+		interface ISelection {
+			buttons: Array<string>;
+			name: string;
+		}
+
+		const textSelection: ISelection = toolbars.styles?.selections?.find(
+			(selection: ISelection) => selection.name === 'text'
+		);
+
+		if (textSelection.buttons) {
+			textSelection.buttons.push('video');
```

Comment:
`AI Creator` plugin does not work with AE. This is unrelated to CXs and the scope of this task, so I chose a simpler existing button to add.

Reply:
I think it'd be clearer if we separate the samples, one for AE and the other for CKEditor. This way, each would make its own assumptions about the internal structure of the `config` object.

However, we don't want to promote AE usage, that is, adding a separate sample for AE looks "too much" advertisement. 

From the  PoV of the implementor, making this distinction in the same sample may be useful in some use cases (e.g. add the same button throughout the entire set of DXP editors).

It's a little bit more cumbersome to justify adding different buttons to different editors, in the same sample. The "video" button is however already present in the base toolbar for ckeditor so we can't change the sample to add it to CKEditor :(

All that said, we can reorganize this sample later on if we find a better way, we can move on with this.

Reply:
> From the PoV of the implementor, making this distinction in the same sample may be useful in some use cases (e.g. add the same button throughout the entire set of DXP editors).

Yeah, that was my line of thinking: this highlights the fact that the same CX can be applied to both Alloy Editor and CKEditors. You need to jump through few hoops, i.e. deal with different config formats, but it's possible. But, I see your point about simplicity, we are trading off simplicity to achieve this.